hardware_layer/hardware_arduino/firmware/arduino_node.cpp

```diff
@@ -0,0 +1,183 @@
+#include "ros.h"
+#include <Arduino.h>
+#include <std_msgs/Int32.h>
+#include <std_msgs/Float64.h>
+#include <math.h>
+#include <Wire.h>
+#include "../include/MS5837.h"
+#include "../include/arduino_thrust.h"
+
+#define pwmPinWest 3
+#define pwmPinEast 2
+#define directionPinEast1 30
+#define directionPinEast2 31
+#define directionPinWest1 32
+#define directionPinWest2 33
+
+#define pwmPinNorthSway 5
+#define pwmPinSouthSway 4
+#define directionPinSouthSway1 27
+#define directionPinSouthSway2 26
+#define directionPinNorthSway1 29
+#define directionPinNorthSway2 287
+
+#define pwmPinNorthUp 6
+#define pwmPinSouthUp 7
+#define directionPinNorthUp1 24
+#define directionPinNorthUp2 25
+#define directionPinSouthUp1 22
+#define directionPinSouthUp2 23
+
+#define analogPinPressureSensor A0
+
+MS5837 sensor;
+
+arduino_thrust AEAST,AWEST,ANORTHUP,ASOUTHUP,ANORTHSWAY,ASOUTHSWAY;
+
+float last_pressure_sensor_value, pressure_sensor_value;
+std_msgs::Float64 voltage;
+ros::NodeHandle nh;
+
+void TEastCb(const std_msgs::Int32 msg)
```

Comment:
Put function definitions later. Over here just put the function declaration

Reply:
Done